src/FlanCommand.ts

```diff
@@ -57,39 +58,86 @@ export default abstract class FlanCommand extends Command {
   }
 
   async loadConfigFile(configPath: string) {
-    let configJSON;
+    // get envirment variable configurations
+    let envConfig: {
+      baseDir?: string;
+      saveDir?: string;
+      repoDir?: string;
+      repository?: string;
+    } = {};
+    let envDbConfig: { host?: string; db?: string; user?: string } = {};
+
+    if (process.env.FLAN_DB_HOST) {
```

Comment:
it's a bit less code to just use process.env.FLAN_DB_HOST directly like `process.env.FLAN_BASE_DIR || configJSON.baseDir || this.localConfig.baseDir`